### PR TITLE
fix(gh-pages): add new favicon icons

### LINKED ISSUE #1533: [🐛 in gh-pages]: add favicon.svg or image 120 × 120 to the favicon list
https://yandex.ru/support/webmaster/search-results/create-favicon.html

### PR DESCRIPTION
**Pull request by HenadzV**
fix(gh-pages): add new favicon svg and png

Added new favicon files to the site assets for better compatibility with modern browsers.

Closes #1533.

### CODE REVIEW

File: pages/views/_layouts/base.njk [hunk 1]
```diff
@@ -14,7 +14,9 @@
   {# Additional site meta information #}
   {% block meta %}{% endblock meta %}
   {# Site icon #}
+  <link rel="icon" type="image/x-icon" href="{{ '/assets/logo-light-blue.svg' | url }}" />
   <link rel="icon" type="image/png" href="{{ '/assets/logo-64x64.png' | url }}" />
+  <link rel="icon" type="image/png" href="{{ '/assets/logo-120x120.png' | url }}" />
```
Comment (ala-n) [inline pages/views/_layouts/base.njk:19]: Not sure about placing icons without any definitions regarding their params...
Is it according to the favicons guide?

**Review by ala-n — CHANGES_REQUESTED** [review]
It seems to be we need to update favicon a little bit more complex 
That [article](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs) seems relevant from what I remember and saw in the guidelines.

**Review by ala-n — APPROVED** [review]

File: pages/static/tools/manifest.webmanifest [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "icons": [
```
Comment (dshovchko) [inline pages/static/tools/manifest.webmanifest:2]: I'm not sure, but I think the `name` is a required field for the manifest. Or am I wrong?

**Review by dshovchko — APPROVED** [review]

File: pages/static/tools/manifest.webmanifest (hunk 2 above)
Comment (ala-n) [reply pages/static/tools/manifest.webmanifest:2]: Good point.
Not sure about general recommendations. But we are definitely going to add more manifest info in the future.
I think that it's ok for the current PR to have just icons to be defined.

**Review by NastaLeo — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

**Review by Natalie-Smirnova — APPROVED** [review]

**Pull request merged**